src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -270,6 +279,51 @@ export default {
 		};
 	},
 	computed: {
+		loanCallouts() {
+			const callouts = [];
+			const activityName = this.loan?.activity.name;
+			const sectorName = this.loan?.sector.name;
+			const tags = this.loan?.tags.filter(tag => tag.charAt(0) === '#')
+				.map(tag => tag.substring(1)) ?? [];
+			const themes = this.loan?.themes ?? [];
+			const categories = {
+				ecoFriendly: tags.includes('Eco-friendly') || tags.includes('Sustainable Ag'),
+				refugeesIdps: themes.includes('Refugees/Displaced'),
+				singleParents: tags.includes('Single Parent')
+			};
+
+			// P1 Category
+			// Exp limited to: Eco-friendly, Refugees and IDPs, Single Parents
+			if (!this.categoryPageName) {
+				if (categories.ecoFriendly) {
+					callouts.push('Eco-friendly');
+				} else if (categories.refugeesIdps) {
+					callouts.push('Refugees and IDPs');
+				} else if (categories.singleParents) {
+					callouts.push('Single Parent');
+				}
+			}
+
+			// P2 Activity
+			if (this.categoryPageName !== activityName) {
+				callouts.push(activityName);
+			}
+
+			// P3 Sector
+			if (sectorName
+			&& (activityName !== sectorName)
+			&& (sectorName !== this.categoryPageName)
+			&& callouts.length < 2) {
+				callouts.push(sectorName);
+			}
+
+			// P4 Tag
+			if (tags && callouts.length < 2) {
+				callouts.push(tags[0]);
```

Comment:
Before pushing tags, we also need to check against the category, otherwise a tag that's actually the category will be pushed here.
![image](https://user-images.githubusercontent.com/16867161/217403189-22a32200-0c79-4e1c-8668-1312cf5816a4.png)

Reply:
The ticket also calls out pulling the tag by random, instead of the first.

Reply:
ooh did not even know that categories and tags can be the same, so much intersection!! :')